src/DynamoCore/DynamoCore.csproj

```diff
@@ -342,6 +352,10 @@ limitations under the License.
       <Project>{6e0a079e-85f1-45a1-ad5b-9855e4344809}</Project>
       <Name>Units</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Libraries\VMDataBridge\VMDataBridge.csproj">
```

Comment:
Is this a cyclic dependency?

Reply:
Both DynamoCore and VMDataBridge use ProtoCore, but VMDataBridge does not use DynamoCore. I did not really like putting this reference into DynamoCore as it is in Libraries, but using the VMDataBridge was required. Possibly VMDataBridge should be moved into core, or maybe there is another way to use it in DynamoCore that I am unaware of.

Reply:
As per discussion after stand-up today this seems ok for now, leaving it as is.